src/cascadia/TerminalApp/CommandPalette.xaml

```diff
@@ -153,12 +176,21 @@ the MIT License. See LICENSE in the project root for license information. -->
                                        HorizontalAlignment="Left"
                                        Text="{x:Bind Name, Mode=OneWay}" />
 
-                            <TextBlock Grid.Column="2"
-                                       Foreground="{ThemeResource SystemControlForegroundBaseMediumBrush}"
-                                       FontSize="12"
-                                       HorizontalAlignment="Right"
-                                       VerticalAlignment="Center"
-                                       Text="{x:Bind KeyChordText, Mode=OneWay}" />
+
+                            <Grid Grid.Column="2"
+                                  Background="{ThemeResource SystemControlForegroundBaseLowBrush}"
+                                  BorderBrush="{ThemeResource SystemControlForegroundBaseMediumBrush}"
```

Comment:
make sure these colors look correct in HC mode

Reply:
It's honestly not too bad
![image](https://user-images.githubusercontent.com/18356694/85862544-88188e00-b787-11ea-9d8f-28665c83e288.png)